feeds/package.json

```diff
@@ -31,6 +31,7 @@
   },
   "dependencies": {
     "antd": "^3.23.3",
+    "bignumber.js": "^9.0.0",
```

Comment:
What do you think about using ethersjs bignum instead?

Reply:
Does it support arbitrary precision decimals? I was under the impression that it's only for integer arithmetic.

Reply:
Unfortunately no, Ethers uses `bn.js`. 
How about this: `ethers.utils.formatUnits(ethers.utils.parseUnits("0.01").mul(100), 18)`

Reply:
I like that's more confusing as percentages are naturally arbitrary precision. I had also considered just using the float in percentage base. i.e.

`0.005` -> `0.5`